metricbeat/module/aerospike/namespace/namespace_integration_test.go

```diff
@@ -5,11 +5,14 @@ package namespace
 import (
 	"testing"
 
+	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 	"github.com/elastic/beats/metricbeat/module/aerospike"
 )
 
 func TestData(t *testing.T) {
+	defer compose.EnsureUp(t, "aerospike")()
```

Comment:
I was first confused by this as I was "should up not be run first"? But now I see that `EnsureUp` returns a function which is executed at the end and `EnsureUp` already here, right?